src/index.html

```diff
@@ -11,6 +11,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      data-qa="big"
+      >
+      <label for="inputLarge" class="search-form">
+        <input
+        type="search"
+        name="inputLarge"
+        id="inputLarge"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="search-form__input"
+        >
```

Comment:
```suggestion
        <input
          type="search"
          name="inputLarge"
          id="inputLarge"
          data-qa="keypress"
          placeholder="Try &quot;Los Angeles&quot;"
          class="search-form__input"
        >
```
pay attention to indentation - fix it everywhere
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/86923089/5ba38ebc-c46c-4fcb-9a87-a35369516694)
see the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)